data-plane/config/100-config-kafka-broker-data-plane.yaml

```diff
@@ -113,3 +113,4 @@ data:
     # ssl.trustmanager.algorithm
   config-kafka-broker-webclient.properties: |
     idleTimeout=10000
+  config-kafka-broker-httpserver.properties: |
```

Comment:
Can you add `idleTimeout` or some default configurations? For example, `idleTimeout` is set to `0` by default which means there is no timeout.

Reply:
TBH I'm in no position to say if it's a good config or not the default idle timeout, so for "demonstration purpose" I've added the idleTimeout conf as default (to check if the properties works)